cli/Makefile

```diff
@@ -203,7 +203,7 @@ snapshot-turbo:
 
 
 .PHONY: publish-turbo
-publish-turbo:
+publish-turbo: clean build
```

Comment:
These deps where missed when this new task was created (see the [previous task](https://github.com/vercel/turbo/blob/tomknickman/turbo-633-new-release-process-is-not-building-js/cli/Makefile#L159) for comparison)